example_files/visualization/input_visualization_feature.html

```diff
@@ -543,13 +544,21 @@ <h2>Annual End Uses</h2>
               values: _.map(months, function (month, i) {
                 var value = 0;
                  if (changeToKbtu){
-                  value = feature.monthly_values['Electricity:Facility'][i]*3.41 - feature.monthly_values['REopt:ElectricityProduced:Total'][i]*3.41;
+                  value = feature.monthly_values['Electricity:Facility'][i]*3.41 + feature.monthly_values['NaturalGas:Facility'][i];
```

Comment:
I think we should include Propane:Facility, FuelOilNo2:Facility and OtherFuels:Facility as well

Reply:
@tanushree04, can you make that change? thanks!